source/reference/minio-mc-admin/mc-admin-user-svcacct-add.rst

```diff
@@ -109,6 +109,22 @@ Parameters
    Add a description for the service account.
    For example, you might specify the reason the service account exists.
 
+.. mc-cmd:: --expiry
+   :optional:
+
+   .. versionadded:: RELEASE.2023-05-30T22-41-38Z
+
+   Set an expiration date for the service account.
+   The date must be in the future, you may not set an expiration date that has already passed.
+
+   Allowed date and time formats:
```

Comment:
As per Kaan they are all RFC-3339 compliant formats so we could maybe state

```suggestion
Specify an :rf:`RFC3339-compliant date-time format <3339>`, such as one of the following:
```

Reply:
It only accepts the ones shown, two of which happen to be RFC 3339. Bummer, I know.